python/perf-kernels/tune_gemm/one_config.py

```diff
@@ -33,28 +35,22 @@ def parse_args():
     parser.add_argument("--waves_per_eu", type=int, default=0)
     parser.add_argument("--matrix_instr_nonkdim", type=int, default=0)
     parser.add_argument("--kpack", type=int, default=0)
-    parser.add_argument("--config_str", type=str, default="", help="can take from tune_gemm.py script output, looks like M16_N8_K128_BM64_BN64_BK64_GM1_SK2_nW2_nS0_EU0_kP2_mfma16")
+    parser.add_argument(
+        "--config_str", type=str, default="", help=
+        "can take from tune_gemm.py script output, looks like M16_N8_K128_BM64_BN64_BK64_GM1_SK2_nW2_nS0_EU0_kP2_mfma16"
+    )
     args = parser.parse_args()
 
     return args
 
 
 def parse_config(cfg_str):
     values = cfg_str.split("_")
-    config_name = {"M": "M",
-                   "N": "N",
-                   "K": "K",
-                   "BM": "BLOCK_SIZE_M",
-                   "BN": "BLOCK_SIZE_N",
-                   "BK": "BLOCK_SIZE_K",
-                   "GM": "GROUP_SIZE_M",
-                   "SK": "SPLIT_K",
-                   "nW": "num_warps",
-                   "nS": "num_stages",
-                   "EU": "waves_per_eu",
-                   "kP": "kpack",
-                   "mfma": "matrix_instr_nonkdim"
-                  }
+    config_name = {
+        "M": "M", "N": "N", "K": "K", "BM": "BLOCK_SIZE_M", "BN": "BLOCK_SIZE_N", "BK": "BLOCK_SIZE_K", "GM":
+        "GROUP_SIZE_M", "SK": "SPLIT_K", "nW": "num_warps", "nS": "num_stages", "EU": "waves_per_eu", "kP": "kpack",
+        "mfma": "matrix_instr_nonkdim"
+    }
```

Comment:
For this part, I feel the old version is better for visualizing the list.

Reply:
It's automated `yapf` formatting from Triton's `pre-commit` config, please see https://github.com/ROCm/triton/blob/triton-mlir/.pre-commit-config.yaml#L33.

I'm going to try the solution described in https://github.com/google/yapf?tab=readme-ov-file#why-does-yapf-destroy-my-awesome-formatting, i.e. disabling `yapf` for a piece of code. If it works, it'd be nice if you could catch other `yapf` misbehaviors...

Reply:
yes, it's better if yapf allows us to do it. 
If it doesn't, we can still manually revert the format change for this part. 
This dict is different from other parts because this is how we shorten the name of each tuning parameter and put them together into the kernel name. We (both tune_gemm developers and users) may need to refer to it now and then.

Reply:
Selectively disabling `yapf` worked, see 2cbf137ac1a1e0b0c1556e4d258512cfcb0af5cc. Please let me know if you find any other `yapf` issue so I can use this trick.